js/arena.js

```diff
@@ -0,0 +1,36 @@
+import Gladiator from './gladiator.js';
+
+export default class Arena {
+    constructor(name) {
+        this.name = name;
+        this.gladiators = [];
+    }
+    addGladiator(gladiators) {
+        if (this.gladiators.length < 2) {
+            this.gladiators.unshift(gladiators);
+
+        }
+    }
+
+
+    fight() {
+        if (this.gladiators.length === 2) {
+            const fight1 = this.gladiators[0].weapon;
+            const fight2 = this.gladiators[1].weapon;
+
```

Comment:
nothing happens after this line this is why nothing shows when the length of the gladiator array is 2